src/main/java/seedu/duke/Parser.java

```diff
@@ -336,7 +336,7 @@ public boolean isFutureDate(String dateWatchedString) {
             dateFormat.setLenient(false);
             date = dateFormat.parse(dateWatchedString);
         } catch (ParseException e) {
-            Ui.print("Invalid date format. Use the following format: DD-MM-YYYY.");
+           return true; //just return true and b/c future date and invalid date are both invalid inputs.
```

Comment:
We already have a print statement in AddMedia I believe so this will be redundant

Reply:
also this will really never happen because we check for valid date before this check